### ISSUE TITLE
[BUG] Plugin validation issues detected on Maven 3.9.2

### ISSUE THREAD

**Issue by jpraet** [OpenAPITools/openapi-generator#15780]
Plugin validation issues detected when using the openapi-generator-maven-plugin (6.6.0) with Maven 3.9.2 and -Dmaven.plugin.validation=VERBOSE

```
[WARNING]  * org.openapitools:openapi-generator-maven-plugin:6.6.0
[WARNING]   Plugin issue(s):
[WARNING]    * Plugin depends on the deprecated Maven 2.x compatibility layer, which may not be supported in Maven 4.x
[WARNING]    * Plugin should declare these Maven artifacts in `provided` scope: [org.apache.maven:maven-compat:3.8.6, org.apache.maven:maven-resolver-provider:3.8.6, org.apache.maven:maven-settings-builder:3.8.6, org.apache.maven:maven-repository-metadata:3.8.6, org.apache.maven:maven-core:3.8.6, org.apache.maven:maven-model-builder:3.8.6, org.apache.maven:maven-plugin-api:3.8.6, org.apache.maven:maven-model:3.8.6, org.apache.maven:maven-builder-support:3.8.6, org.apache.maven:maven-settings:3.8.6]
```

**Comment by wing328** [comment]
thanks for opening an issue on this. I noticed something similar when updating maven to the latest version.

Is that something you can help us with? If yes, let us know if you need help opening a PR 🙏

**Comment by rubensa** [comment]
Duplicate from https://github.com/OpenAPITools/openapi-generator/issues/15672

**Comment by Philzen** [comment]
@jpraet This seems to have been fixed via #16274.

Can you check if this issue can be closed?

**Comment by jpraet** [comment]
Yes, it is fixed.
